polar-core/src/polar.rs

```diff
@@ -127,6 +128,7 @@ impl Iterator for Query {
 }
 
 const ROLES_POLICY: &str = include_str!("roles.polar");
+const VALIDATE_ROLES_CONFIG_RESOURCES: &str = "resource(resource, name, permissions, roles)";
```

Comment:
to match the variable names we use in the predicate definition

```suggestion
const VALIDATE_ROLES_CONFIG_RESOURCES: &str = "resource(resource, name, actions, roles)";
```